tests/EmailReplyParser/Tests/Parser/EmailParserTest.php

```diff
@@ -273,6 +273,22 @@ public function testEmailGmailNo()
         $this->assertEquals(static::COMMON_FIRST_FRAGMENT, trim($fragments[0]));
     }
 
+    public function testEmailFreeZimbraFr()
+    {
+        $email     = $this->parser->parse($this->getFixtures('email_zimbra_free_fr.txt'));
+        $fragments = $email->getFragments();
+        $this->assertStringContainsString('Michael Scott', $fragments[0]);
+        $this->assertStringNotContainsString('Toby Flenderson', $fragments[0]);
+    }
+
+    /*public function testEmailFreeZimbraEn()
```

Comment:
Please remove this. We don't need commented code.

Reply:
I just commented it because it fails, so I think it should be uncommented and the regex should be fixed. In current state, that kind of email is wrongly parsed, and data can be lost.
But as I said, I don't understand all EN tests and rules, so I didn't want to do it myself. However, I can remove this indeed, I don't have EN users anyway.

Reply:
Well, unless you find a way to fix the regex without breaking other test cases, I don't think it's very useful to keep a commented test case.